doc/api/http.md

```diff
@@ -333,9 +333,8 @@ Until the data is consumed, the `'end'` event will not fire. Also, until
 the data is read it will consume memory that can eventually lead to a
 'process out of memory' error.
 
-Unlike the `request` object, if the response closes prematurely, the
-`response` object does not emit an `'error'` event but instead emits the
-`'aborted'` event.
+For backward compatibility, `res` will only emit `'error'` if there is an
+`'error'` listener registered.
```

Comment:
Should the `aborted` part not be kept? If I understood correct, that's still the case if there's no `error` listener attached.

Reply:
I'm not sure, the previous paragraph was there because we didn't emit `'error'`. Since this is only informative I would rather not mention something that we would like to deprecate in the future.